sections/custom-fields.md

```diff
@@ -122,6 +122,8 @@ POST /api/v1/users/<user_id>/custom_field_values
 | `user_id` or `project_id` | number | `id` of the associated user or assignable (can be inferred from URL) |
 | `value` | string or array | the actual value |
 
+:warning: **Note**: Only multi-select custom fields (i.e., those with a `data_type` of `3`) can accept an array as a value in a `POST` request. This will create one custom field value record for each value in the array. Note that subsequent `POST` requests will overwrite the current values. If you wish to append to an array of custom field values associated with a custom field, it is necessary to `POST` the entire array (i.e., the current array values, plus any additions). Similarly, if you wish you delete certain values, you must post the desired final values.
```

Comment:
How about

> **Note**: Only custom fields with a `data_type` of `multiple_choice_selection_list` can accept an array with multiple values in a `POST` request. Note that this array is treated as the complete set of values for the given custom field and project/user. To delete values, you can do so through another `POST` request in which only the desired values are contained in the array, and the unwanted values are absent from the array.

Reply:
It's this sentence that seems a little ambiguous.

> Note that this array is treated as the complete set of values for the given custom field and project/user. 

Because you then describe the delete case, and leave the append case to inference.

Reply:
What about

> **Note**: Only custom fields with a `data_type` of `multiple_choice_selection_list` can accept an array with multiple values as its `value` in a `POST` request. Note that this array is treated as the complete set of values for the given custom field and project/user. Subsequent `POST` requests for the same custom field and project/user will result in the deletion of previous values if those values are not also included in the `value` array. In other words, subsequent `POST` requests do not _append_ new values, but rather _replace_ the _set_ of values as a whole.